### DISCUSSION TITLE
Login with my-org.login.domain.com gives me  ID=QUERY-n0wng Message=Instance not found

### DISCUSSION THREAD

**Discussion by itxworks** [zitadel/zitadel#6666] [General]
Hello, 

I am not sure how this works - or should work on self hosted with docker and traefik

I am using:

ZITADEL_EXTERNALPORT=443
ZITADEL_EXTERNALDOMAIN=login.domain.com

So when I add an additional org-domain like my-org -> the login url should be my-org.login.domain.com right?

But I get:  ID=QUERY-n0wng Message=Instance not found

Thank you.

**Comment by mffap** [comment]
Your custom domain will still be `login.domain.com`. You can send a custom scope to force an organizations policies or use domain discovery to do so based on the username.

org scope: https://zitadel.com/docs/apis/openidoauth/authrequest#organization-policies-and-branding

domain discovery: https://zitadel.com/docs/guides/solution-scenarios/domain-discovery

**Reply by itxworks** [reply]
OK - thank you.

In my test case I do have 2 orgs - lets call them internal and external.

I do like to let user to self register with the external - but not  with the internal (disabled).

Maybe I did something wrong - but this only works if I set external as "primary".

Because I would like to accept free email accounts like gmail, gmx, proton, etc. ... 

Is there any way to setup the the org-domain within the register process or url (like keycloak realms) by not using it like user@my-org.login.domain.com 

Thank's
